### PR TITLE
Skip connected Stripe and PayPal rows during developer dump

### PR DESCRIPTION
**Pull request by gregorbg**
Hotfix as per call.

### CODE REVIEW

**Review by dunkOnIT — APPROVED** [review]
My most confident LGTM to date!

**Comment by dunkOnIT** [comment]
(Presuming this one was ok to merge - let me know if you'd rather I stay off the merge button in future)

**Comment by gregorbg** [comment]
No worries, totally fine!

**Pull request merged**
